Cargo.lock

```diff
@@ -4554,6 +4556,7 @@ dependencies = [
  "pallet-balances",
  "pallet-currency-factory",
  "pallet-instrumental",
+ "pallet-pablo",
```

Comment:
As I [said earlier](https://github.com/ComposableFi/composable/pull/1130#discussion_r893555349) after each update of `Cargo.toml` `Cargo.lock` file needs to be updated too. `pallet-pablo` and `composable-maths` should be removed after rebuilding (rechecking, etc.).